packages/genesys-spark-components/src/components/stable/gux-popover-list/gux-popover-list.tsx

```diff
@@ -205,6 +205,8 @@ export class GuxPopoverList {
           'gux-hidden': !this.isOpen,
           'gux-popover-wrapper': true
         }}
+        role="dialog"
+        aria-describedby="gux-popover-content"
```

Comment:
`aria-describedby` takes a list of ids not a class. You will have to give the element an auto generate id and then reference it here.

Reply:
Fixed